### PR TITLE
fix: [taskdialog]Every time a small file is copied, the position of the copy window will be refreshed

### PR DESCRIPTION
**Pull request by liyigang1**
The size of each item will be adjusted when the height of the copy progress bar is changed. Here, the position of the entire display dialog box is adjusted. Modify without adjusting the entire display dialogue position.

Log: Every time a small file is copied, the position of the copy window will be refreshed
Bug: https://pms.uniontech.com/bug-view-199201.html https://pms.uniontech.com/bug-view-199209.html

### CODE REVIEW

**Review by Lighto-Ku — DISMISSED** [review]

File: src/dfm-base/dialogs/taskdialog/taskdialog.cpp [hunk 1]
```diff
@@ -177,7 +184,9 @@ void TaskDialog::adjustSize()
     }
 
     layout()->setSizeConstraint(QLayout::SetNoConstraint);
-    moveYCenter();
```
Comment (Johnson-zs) [inline src/dfm-base/dialogs/taskdialog/taskdialog.cpp:180]: adjustSize和moveToCenter是两件事情，为何要写到一个接口中呢，外部根据需要显式的直接调用moveToCenter不行吗

**Review by Kakueeen — DISMISSED** [review]

File: src/dfm-base/dialogs/taskdialog/taskdialog.cpp [hunk 2]
```diff
@@ -150,11 +150,18 @@ void TaskDialog::setTitle(int taskCount)
...
+}
 
 /*!
  * \brief TaskDialog::adjustSize 调整整个进度显示的高度，当每个item中的widget发生变化时
  */
-void TaskDialog::adjustSize()
+void TaskDialog::adjustSize(const bool moveToYcenter)
 {
     if (!adjustSizeMutex)
         adjustSizeMutex = new QMutex();
```
Comment (Johnson-zs) [inline src/dfm-base/dialogs/taskdialog/taskdialog.cpp]: TaskDialog都在主线程中应用，有必要加锁吗？可以直接构造函数断言在主线程中构造

File: src/dfm-base/dialogs/taskdialog/taskdialog.cpp (hunk 1 above)
Comment (liyigang1) [reply src/dfm-base/dialogs/taskdialog/taskdialog.cpp:180]: 这是以前的代码，没有做过修改。好的立刻修改

File: src/dfm-base/dialogs/taskdialog/taskdialog.cpp (hunk 2 above)
Comment (liyigang1) [reply src/dfm-base/dialogs/taskdialog/taskdialog.cpp]: 以前的代码，没有修改。立刻修改

**Review by max-lvs — COMMENTED** [review]
提交到 master 分支

**Pull request closed without merging**